code/src/library/Containers.js

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components'
+
+export const Main = styled.main`
+${({noList}) => noList?`height: 100vh;`:`height:100%;`}
+background-image: linear-gradient(45deg, #4e6048, #70836d, #93a895, #b8cebf);
+display: grid;
+grid-template-column: 1fr;
+grid-template-rows: min-content;
+justify-content: center;
+padding: 10px;
+`
+
+export const Wrapper = styled.section`
```

Comment:
Here you are using both flex and grid in the same styled component. You are setting width to 100% and max-width to 90%. 
Depending if  I have todos in my list or no todos in the list the wrappers size differ. If I add a lot of todos due to height of 100vh in main the list will not fit the main wrapper.